librz/include/rz_util/rz_strbuf.h

```diff
@@ -12,7 +12,6 @@ typedef struct {
 	size_t len; // string length in chars or binary buffer size
 	char *ptr; // ptr replacing buf in case strlen > sizeof(buf)
 	size_t ptrlen; // string length + 1 or binary buffer size
-	bool weakref; // ptr is not owned
```

Comment:
@XVilka this can't be done in a patch release, in theory. It should be done in the next release.

Reply:
Good point, will remove that commit